common/db.go

```diff
@@ -201,6 +215,39 @@ func InitDB(dbPath string) (*DB, error) {
 	}
 	d.insertWinningTicket = stmt
 
+	// Insert block header
+	stmt, err = db.Prepare("INSERT INTO blockheaders(number, parent, hash, logs) VALUES(?, ?, ?, ?)")
+	if err != nil {
+		glog.Error("Unable to prepare insertMiniHeader", err)
```

Comment:
Missing `d.Close()` in the error handlers for all of the statement preparations except for `deleteMiniHeader`

Reply:
Fixed in ced063e